opentabs.py

```diff
@@ -138,48 +137,32 @@ def run(self):
           on_highlight = self.when_file_selected
         )
 
-  def add_group_views(self):
-    self.has_groups = True
+  def add_views(self):
     window = self.window
     folder_name = self.get_folder_name()
     groups = list(range(window.num_groups()))
     for group_index in groups:
       views = window.views_in_group(group_index)
 
       for view in views:
+
+        self.index += 1 #start at 0
```

Comment:
this whole block is kind of meaningless if you have 1 group, which is why I separated out the methods for creating views in my version

Reply:
wrapped the code in a if 

```python
if not self.has_group:
  ....
```